settings/security/third-party/saml.rst

```diff
@@ -23,6 +23,10 @@ Connect your SAML identity provider as a single sign-on (SSO) method.
    https://docs.microsoft.com/en-us/windows-server/identity/active-directory-federation-services
 .. _Okta: https://www.okta.com/
 
+.. warning:: We currently only provide basic instructions for connecting Zammad
+   with Keycloak. Other IdPs should be configured similar, but we can't provide
+   instructions for other IdPs.
+
```

Comment:
I suggest to just say `Please note: Our instructions are based on connecting Zammad with Keycloak.` or something similar.